LICENSE

```diff
@@ -850,6 +850,61 @@ FROM, OUT OF THE USE OR INABILITY TO USE THE FONT SOFTWARE OR FROM
 OTHER DEALINGS IN THE FONT SOFTWARE.
 
 
+for share/server/60/esprima.js
```

Comment:
Looking good.

We still need to place a copy of these upstream dependencies untouched into an Apache repository, as @janl said on IRC. If we made modifications, we should track those in a branch in that repo as well. If we can't solve the build problem now (how to generate these files from those repos), please open a new ticket and mention Jan and Garren in it so we can look at fixing that for the future.

I want to be sure that if these libraries change in the future, we can easily absorb the fixes and improvements into our code, _especially_ if they're security-related patches.

@davisp can help you with the repo importation into Apache, he's done it recently for part of the 4.0 libraries.

Reply:
Thanks for your suggestions, @wohali I created https://github.com/apache/couchdb/issues/2372 for now.

Reply:
@wohali @jiangphcn Yap, will do that in a few minute after I finish testing this PR.